roles/telemetry_logging/README.md

```diff
@@ -0,0 +1,42 @@
+telemetry_logging
+=========
+
+Test logging in Openstack
+
+Requirements
+------------
+The following resources are required in the OpenStack cloud
+* network called public
+* network called provate
+* security group called basic with ssh and tcp enabled to the VMs
+* a flavor called m1.small
+* an image called cirros
+
```

Comment:
These are not required for the logging role as far as I know.

They are required for the telemetry_autoscaling role because the heat templates need these resources to exist.